tests/Returns/Http/Requests/ReturnRequestTest.php

```diff
@@ -0,0 +1,142 @@
+                    "first_name" => $faker->firstName(),
+                    "last_name" => $faker->lastName(),
+                    "email" => $faker->email(),
+                    "phone_number" => $faker->phoneNumber(),
+                ],
+                'return_address'   => [
+                    "street_1" => $faker->streetName(),
+                    "street_2" => $faker->streetName(),
+                    "street_number" => $faker->randomNumber(),
+                    "street_number_suffix" => $faker->word(),
+                    "postal_code" => $faker->postcode(),
+                    "city" => $faker->city(),
+                    "state_code" => $faker->countryCode(),
+                    "country_code" => $faker->countryCode(),
+                    "company" => $faker->company(),
+                    "first_name" => $faker->firstName(),
+                    "last_name" => $faker->lastName(),
+                    "email" => $faker->email(),
+                    "phone_number" => $faker->phoneNumber(),
+                ],
+                'payment'          => [
+                    'external_payment_id' => 'payment-12345',
+                    'amount'              => 1000,
+                    'currency'            => 'USD',
+                ],
+                'items'            => [
+                    [
+                        'external_reference' => 'ref-item-1',
+                        'sku'                => 'sku-123',
+                        'name'               => 'Sample Item 1',
+                        'quantity'           => 2,
+                        'price_amount'       => 500,
+                        'currency'           => 'USD',
+                        'weight'             => 10,
+                        'weight_unit'        => 'kg',
+                        'comment'            => 'No issues',
+                        'return_reason'      => 'damaged',
+                        'description'        => 'A sample item description',
+                    ],
+                    [
+                        'external_reference' => 'ref-item-2',
+                        'sku'                => 'sku-456',
+                        'name'               => 'Sample Item 2',
+                        'quantity'           => 1,
+                        'price_amount'       => 300,
+                        'currency'           => 'USD',
+                        'weight'             => 20,
+                        'weight_unit'        => 'kg',
+                        'comment'            => 'Wrong size',
+                        'return_reason'      => 'wrong_size',
+                        'description'        => 'Another sample item description',
+                    ],
+                ]
+            ]
+        ];
+
+        $request = new ReturnRequest();
+        $request->replace($stub);
+
+        $this->assertEquals($request->orderReference(), Arr::get($stub, 'data.order_reference'));
```

Comment:
In these assertions the arguments should be swapped, first is the exepcted result, then is the evaluation. Can you check other test files if they are having the same?